_layouts/tutorial.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: basic
+SPDX-License-Identifier: CC-BY-SA-4.0
+SPDX-FileCopyrightText: 2020 seL4 Project a Series of LF Projects, LLC.
+---
+
+<div class="row">
+  <div class="hidden-xs col-sm-4 col-md-3 col-lg-2">
+    {% include nav-sidebar.html %}
+
+  </div>
+  <div class="content col-sm-8 col-md-6 col-lg-7 main">
+    {{ content }}
+  </div>
+
+{% assign url = page.url | split: "/" %}
+{% capture project_name %}
+{% if page.project %}
+{{page.project}}
+{% elsif url[1] == "projects" %}
+{{url[2]}}
+{% endif %}
+{% endcapture %}
+
+{% assign project_name = project_name | strip %}
+{% assign project = site.data.projects[project_name] %}
+
+  {% if page.toc %}
+    {% include toc-sidebar.html %}
+  {% endif %}
+</div>
+<script src="{{ base.url | prepend: site.url }}/assets/js/toggle-markdown.js"></script>
```

Comment:
This should be slightly more stable when we change jekyll versions:
```suggestion
<script src="{{ '/assets/js/toggle-markdown.js' | relative_url }}"></script>
```
(`relative_url` produces an absolute path from the site root taking into account any base url and other config settings Jekyll is applying)